csi/volumeutils.py

```diff
@@ -1019,7 +1023,7 @@ def mount_glusterfs(volume, mountpoint, is_client=False):
                 "-oStrictHostKeyChecking=no",
                 "-i",
                 "%s" % secret_private_key,
-                "%s@%s" % (secret_username, volume['g_host']),
+                "%s@%s" % (secret_username, g_host),
```

Comment:
it is used in another place also? during volume expansion. Can you check if thats handled too?

Also mention Issue Reference in commit message.

Reply:
Done